cryptographic-properties-1.1.schema.json

```diff
@@ -499,6 +569,33 @@
             "title": "ConfidenceLevel",
             "description": "The ConfidenceLevel related to a property",
             "type": "number"
+        },
+        "securedBy": {
+            "type": "object",
+            "title": "Crypto asset is secured by",
+            "description": "Specifies the mechanism by which the crypto asset is secured by",
+            "additionalProperties": false,
+            "properties": {
+                "mechanism": {
+                    "type": "string",
+                    "title": "mechanism by which the crypto asset is secured by",
+                    "description": "Specifies the mechanism by which the crypto asset is secured by",
+                    "examples": [
```

Comment:
what if the asset is not secured? perhaps add "NotSecured" as an example to make it explicit.